lib/Doctrine/ORM/Query/Parser.php

```diff
@@ -1464,7 +1464,7 @@ public function GroupByItem()
         // We need to check if we are in a IdentificationVariable or SingleValuedPathExpression
         $glimpse = $this->lexer->glimpse();
```

Comment:
Looks like we're missing `|null` in the return of this method too, could you please check it?

Reply:
> this method too

Do you mean `Parser::GroupByItem()` or `AbstractLexer::glimpse()` (that already has `@return array|null`)?

Reply:
I meant the `AbstractLexer::glimpse()` because https://scrutinizer-ci.com/g/doctrine/orm/inspections/38753df0-3b11-4ffd-88ab-49eedf907e23/issues/files/lib/Doctrine/ORM/Query/Parser.php?status=new&orderField=path&order=asc&honorSelectedPaths=0 but it might be that scrutinizer is just tripping.